projects/jsoncpp/build.sh

```diff
@@ -21,10 +21,12 @@ cmake -DCMAKE_CXX_COMPILER=$CXX -DCMAKE_CXX_FLAGS="$CXXFLAGS" \
       -DBUILD_SHARED_LIBS=OFF -G "Unix Makefiles" ..
 make
 
-# Compile fuzzer.
-$CXX $CXXFLAGS -I../include $LIB_FUZZING_ENGINE \
-    ../src/test_lib_json/fuzz.cpp -o $OUT/jsoncpp_fuzzer \
-    lib/libjsoncpp.a
+rm -rf genfiles && mkdir genfiles && ../LPM/external.protobuf/bin/protoc json.proto --cpp_out=genfiles --proto_path=$SRC
 
-# Add dictionary.
-cp $SRC/jsoncpp/src/test_lib_json/fuzz.dict $OUT/jsoncpp_fuzzer.dict
+$CXX $CXXFLAGS -I genfiles -I.. -I ../libprotobuf-mutator/ -I ../LPM/external.protobuf/include -I../include $LIB_FUZZING_ENGINE \
```

Comment:
Is the `-I..` needed? It has odd spacing relative to the other -I lines (although consistent with `-I../include` later, but it's unclear which of the new files needs that `#include`

Basically, I'm trying to figure out if:
1. This is influenced by `$SRC` from Line 24
2. This should be `$SRC` itself

Reply:
-I.. is needed. I agree that is has odd spacing, so I did fix that. To answer your question, it should not be $SRC itself and the file that needs that #include is /src/jsoncpp_fuzz_proto.cc. I knew to include this from looking at https://github.com/google/oss-fuzz/blob/master/projects/libpng-proto/build.sh#L47.